microsoft-edge/webview2/release-notes.md

```diff
@@ -42,22 +42,39 @@ Release Date: September 6, 2022
 
 For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 105.0.1343.22 or higher.
 
+### General
+
+This WebView2 SDK release has the same bug fixes that are in WebView2 SDK 1.0.1369-prerelease.
+
+#### Bug fixes

```

Comment:
We can remove the bug fixes listed here since they are mentioned below as well

Reply:
Removed this Bug Fixes section and pointed to the below Bug Fixes section.